### PR TITLE
add method: Model.set_tree_limit

### PR DESCRIPTION
**Pull request by sperlingxx**

### CODE REVIEW

**Comment by sperlingxx** [comment]
Hi @hcho3 , can you help to review this :)

File: python/setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def has_ext_modules(self):
 LIB_PATH = LIBPATH['find_lib_path'](basename='treelite')
 RT_PATH = LIBPATH['find_lib_path'](basename='treelite_runtime')
 
-if (not LIB_PATH) or (not RT_PATH) or (not os.path.isdir('../build/runtime')):
```
Comment (hcho3) [inline python/setup.py]: Why are we changing this line?

Comment (sperlingxx) [reply python/setup.py]: Because I don't have `../build/runtime`, and I also setup python wrapper successfully.

Comment (hcho3) [reply python/setup.py]: Can you check if you can import `treelite.runtime` in your Python shell?

Comment (sperlingxx) [reply python/setup.py]: I have rebuilt TreeLite on a Docker image.
1.  mkdir build && cd build && cmake .. && make -j4
2. rm -rf runtime 
3. cd ../python && python setup.py (remove `os.path.isdir('../build/runtime')`) install

After above steps, I found no exceptions thrown when I ran `import treelite.runtime` in Python shell.

Comment (sperlingxx) [reply python/setup.py]: I reverted it anyway.

**Pull request merged**